sw/tools/install.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+import sys
+from PyQt5.QtCore import *
+from PyQt5.QtGui import *
+from PyQt5.QtWidgets import *
+
+import lsb_release
+import subprocess
+import webbrowser
+
+release = lsb_release.get_distro_information()
+docs = 'https://paparazzi-uav.readthedocs.io'
+
+
+
+def execute(cmd):
+    print(cmd)
+    p = subprocess.Popen(cmd, shell=True)
+    p.communicate() # wait until ready
+    print('================================================================================')
+    print(cmd)
+    print('READY (verify above)')
+    print('Check documentation to resolve problems:', docs)
+    print('================================================================================')
+
+
+def view(link):
+    webbrowser.open(link)
+
+def cmd_repo():
+    execute('sudo add-apt-repository -y ppa:paparazzi-uav/ppa && sudo apt-get update')
+
+
+def cmd_dev():
+    execute('sudo apt-get -f -y install paparazzi-dev')
+    # Missing
+    if float(release['RELEASE']) <= 20.04:
+        execute('sudo apt-get install python3-lxml python3-numpy')
+
+def cmd_arm():
+    execute('sudo apt-get -f -y install paparazzi-dev')
+    if float(release['RELEASE']) >= 20.04:
+        execute('sudo apt-get install python-is-python3 gcc-arm-none-eabi gdb-multiarch')
+        # python3-serial?
+    else:
+        execute('sudo add-apt-repository -y ppa:team-gcc-arm-embedded/ppa')
+        execute('sudo apt-get update')
+        execute('sudo apt-get -f -y install gcc-arm-embedded')
+
+def cmd_jsb():
+    execute('sudo apt-get -f -y install paparazzi-jsbsim')
+
+def cmd_gcs():
+    execute('sudo apt-get -f -y install pprzgcs')
+
+def cmd_mcu():
+    execute('sudo apt-get -f -y install dfu-util')
+    execute('sudo cp conf/system/udev/rules/*.rules /etc/udev/rules.d/ && sudo udevadm control --reload-rules')
+
+def cmd_gazebo():
+    execute('sudo apt-get -f -y install gazebo9 libgazebo9-dev')
+    execute('git submodule init && git submodule sync && git submodule update ./sw/ext/tudelft_gazebo_models')
+
+
+def cmd_bebopcv():
+    execute('git submodule init && git submodule sync && git submodule update ./sw/ext/opencv_bebop')
+    execute('sudo apt-get -f -y install cmake libjpeg-turbo8-dev libpng-dev libtiff-dev zlib1g-dev libdc1394-22-dev')
+
+def cmd_vlc():
+    execute('sudo apt-get -f -y install ffmpeg vlc jstest-gtk default-jre')
+    execute('sudo apt-get install python3-pip')
+    if float(release['RELEASE']) <= 20.04:
+        execute('pip3 install pyquaternion ivy-python') # Required for NatNat
+    else:
+        execute('pip3 install pyquaternion ivy') # Required for NatNat
```

Comment:
Yes. This was reported and I tested it in a new VM myself. Without these extra packages `natnet` from master/release6.2 does not start. in 18.04 pip install ivy does not work as it needs python3.8 so I install `ivy-python` instead and finally `natnet` works. For future `paparazzi-dev` packages, it would be nice to add these missing python3 packages or instead make a `requirements.txt` file such that people with more versions and `conda` have fewer problems.